src/js/index.d.ts

```diff
@@ -34,7 +35,7 @@ declare module "lottie-react-native" {
      * animation will correspondingly update to the frame at that progress value. This
      * prop is not required if you are using the imperative API.
      */
-    progress?: number | Animated.Value;
+    progress?: number | Animated.Value | Reanimated.Node;
```

Comment:
This type does not compile with typesscript since Node require an argument

`Generic type 'AnimatedNode<T>' requires 1 type argument(s).`.

Reply:
@mbret good catch! could you submit a PR to fix it?

Reply:
@emilioicai this pr also forces you to have react-native-reanimated install, as otherwise errors because the module is not resolved. I'm not sure if that was an intended side-effect

Reply:
I think it's possible to change the import to make it conditional, but it requires a little bit more handling. Or maybe we export 2 different components. One that imports reanimated and one that uses the other one.

Reply:
I think you can make it using `require` directly but I'm afraid reanimated would still need to be added as a dependency which may confuse users who doesn't use reanimated on their projects. I'm happy to think this over though as I consider Reanimated is a great tool.